cmd/kg/main.go

```diff
@@ -94,6 +95,7 @@ func Main() error {
 	local := flag.Bool("local", true, "Should Kilo manage routes within a location?")
 	logLevel := flag.String("log-level", logLevelInfo, fmt.Sprintf("Log level to use. Possible values: %s", availableLogLevels))
 	master := flag.String("master", "", "The address of the Kubernetes API server (overrides any value in kubeconfig).")
+	mtu := flag.Uint("mtu", wireguard.DefaultMTU, "The MTU of the WireGuard interface created by Kilo.")
```

Comment:
e2e tests were failing because the Kilo container was crashlooping. This happened because I was passing `"master"` as the first argument, which caused `flag` to panic during flag-parsing.